### DISCUSSION TITLE
How to scale and re-scale with LightningDataModule and MinMaxScaler

### DISCUSSION THREAD

**Discussion by denisbeslic** [Lightning-AI/pytorch-lightning#18724] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
This is a very similar quesiton to https://github.com/Lightning-AI/lightning/discussions/11297, but my comment remained unanswered.

I'm working on a Transformer-based model to predict signal data. The data has to be transformed (standardized) when training/validating and later inverse-transformed when predicting in production.
I'm not sure how to save the scaler parameters during training so that can I use them later during inference:

```
class LitDataModule(LightningDataModule):
    def __init__(self):
        super().__init__()
        self.scaler = MinMaxScaler()
    def setup(self, stage):
        if state == 'fit':
            train_data = self.scaler .fit_transform(train_data)
            val_data =  self.scaler .transform(val_data)
        if state =='test':
            test_data =  self.scaler .transform(test) 
            # This will throw "This MinMaxScaler instance is not fitted yet." exception
            # because we do not save the scalar params during training

```
How can I save the parameters of the scaler during the training mode and load it later during inference mode?

Thanks for your help.
